scripts/seal.coffee

```diff
@@ -0,0 +1,9 @@
+module.exports = (robot) ->
+  robot.hear /seal me up/i, (res) ->
```

Comment:
Good point! I need to get back to this work... I have to make adjustments to the seal too. On my list!